samples/bot-all-cards/csharp/README.md

```diff
@@ -0,0 +1,151 @@
+---
+page_type: sample
+description: Type of Cards.
+products:
+- dotnet
+- office-teams
+languages:
+- csharp
+extensions:
+ contentType: samples
+ createdDate: "21/10/2022 07:54:21 PM"
+urlFragment: officedev-microsoft-teams-samples-bot-all-cards-csharp
+
+---
+
+# Using Cards
+
+Bot Framework v4 using cards bot sample
+
+This bot has been created using [Bot Framework](https://dev.botframework.com), it shows how to create a bot that uses rich cards to enhance your bot design.
+
+## Interaction with app
+
+![all-cards-sample ](BotAllCards/Images/allBotCardsGif.gif)
+
+## Prerequisites
+
+- [.NET Core SDK](https://dotnet.microsoft.com/download) version 6.0
+
+  ```bash
+  # determine dotnet version
+  dotnet --version
+  ```
+- Publicly addressable https url or tunnel such as [ngrok](https://ngrok.com/) or [Tunnel Relay](https://github.com/OfficeDev/microsoft-teams-tunnelrelay)
+
+## Setup
+
```

Comment:
I can't see step for app registration.
Please refer NodeJS PR 
https://github.com/OfficeDev/Microsoft-Teams-Samples/pull/668

Reply:
Fixed